src/main/java/com/hackclub/hccore/advancements/HubAdv.java

```diff
@@ -0,0 +1,23 @@
+package com.hackclub.hccore.advancements;
+
+import com.fren_gor.ultimateAdvancementAPI.advancement.Advancement;
+import com.fren_gor.ultimateAdvancementAPI.advancement.BaseAdvancement;
+import com.fren_gor.ultimateAdvancementAPI.advancement.display.AdvancementDisplay;
+import com.fren_gor.ultimateAdvancementAPI.advancement.display.AdvancementDisplayBuilder;
+import com.fren_gor.ultimateAdvancementAPI.util.AdvancementKey;
+import com.fren_gor.ultimateAdvancementAPI.util.CoordAdapter;
+import com.hackclub.hccore.HCCorePlugin;
+import org.bukkit.Material;
+
+public class HubAdv extends BaseAdvancement {
+    static AdvancementDisplayBuilder<AdvancementDisplay.Builder, AdvancementDisplay> displayBuilder = new AdvancementDisplay.Builder(Material.POWERED_RAIL, "Linked Up")
+            .goalFrame()
+            .announceChat()
+            .showToast()
+            .description("Connect your base to the Nether Hub");
```

Comment:
is this advancement still wanted / needed?

Reply:
We will build a nether hub on the new world (I forgot who but someone's already working on bridges). Maybe it should be disabled until the hub is fully built?

Reply:
Yes, I can make it invisible until its granted. should I move the children of that advancement to somewhere else then? (the children won't show if the parent is hidden, right?)

Reply:
yeah i think so